### ISSUE TITLE
API: Improve DX by adding more named arguments to reason-skia

### ISSUE THREAD

**Issue by lessp** [revery-ui/revery#873]
Since we also expose the Skia-bindings for a Canvas-like API I think it might be a good idea to start refactoring the API to utilise more named arguments e.g. `~x=..., ~y=...`.

What do you others think?

**Comment by glennsl** [comment]
Yes please! I'm working with the Skia/Canvas API right now, and needing to look up what the umpteen different `float` arguments of a function means is definitely a frustrating experience.
